### PR TITLE
added task_01 and task_02

### PR DESCRIPTION
**Pull request by eliz79**
Attempting to completed task 1 & 2

### CODE REVIEW

File: task_01.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! usr/bin/env python
+# *-* coding: utf-8 *-*
+"""Creating a dictionary."""
+
+GRADE_DATA = {
+    'Luke': {'subject': ['math', 'etiquette', 'grammar', 'gym']},
```
Comment (cheuschober) [inline task_01.py]: If you look at the test result detail it shows you the expected construction. Compare what you have here against the output from the failed test and see if a pattern emerges.

File: task_04.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Merging a dictionary."""
+
+from data import BANDS
+
+BANDS['Fleetwood Mac'].update({
+    'Buckingham Nicks': {
```
Comment (cheuschober) [inline task_04.py]: You first needed to add buckingham nicks to the list of bands (Step 2)

File: task_05.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Changing dictionary values."""
+
+from data import SUPERHEROES
+
+SUPERHEROES['Logan'] = {'Wolverine': 'Weapon X'}
```
Comment (cheuschober) [inline task_05.py]: SUPERHEROES['Logan']['alias'] = 'Wolverine'

File: task_07.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        139791890
+
+        >>> import task_07
+        >>> task_07.iter_dict_funky_sum(DICT2)
+        468
+    """
+    funky_total = 0
+    for key, value in myvar.iteritems():
+        funky_total = funky_total + value - key
+    return funky_total
```
Comment (cheuschober) [inline task_07.py]: @eliz79 You did the function correctly but instead declared DICT1 and DICT2 instead of a variable named DATA as directed in step 2!

Comment (eliz79) [inline task_07.py]: awww man

**Pull request merged**